docs/howto/kubernetes/gardener/create-shoot-cluster.md

```diff
@@ -1,68 +1,70 @@
+
+!["Create {{k8s_management_service}} Shoot Cluster" panel](assets/create-shoot-3.png)
+
+In the list of clusters, you will see your new {{k8s_management_service}} shoot bootstrapping.
+The icon on the left marks the progress.
+Creating the cluster can take several minutes.
 
 ![Shoot cluster bootstrapping](assets/shoot_bootstrapping.png)
 
-## Extract kubeconfig file from Shoot cluster
+### A note on quotas
+
+Your Gardener worker nodes are subject to [quotas](../../../reference/quotas/openstack.md) applicable to your {{brand}} project.
+You should make sure that considering your selection of worker node [*flavor*](../../../reference/flavors/index.md) (which determines the number of virtual cores and virtual RAM allocated to each node), the _volume size_, and the _Autoscaler Max_ value, you are not at risk of violating any quota.
 
-When the Shoot cluster is up and running, you need to get a kubeconfig
-file to be able to access it. To do that, click on the cluster to
-expand its properties, and open `Kubeconfig`.
+For example, if your project is configured with the [default quotas](../../../reference/quotas/openstack.md), and you select the `b.4c16gb` flavor for your worker nodes, your cluster would be able to run with a maximum of 3 worker nodes (since their total memory footprint would be 3×16=48 GiB, just short of the default 50 GiB limit).
+A 4th node would push your total memory allocation to 64 GiB, which would violate your quota.
 
-![Kubeconfig in {{k8s_management_service}} Shoot view](assets/shoot_kubeconfig.png)
+If necessary, be sure to request a quota increase via our [{{support}}](https://{{support_domain}}).
 
-You should now see the file's contents. You have the option to `Copy
-Config` or `Rotate Kubeconfig` if your credentials got compromised.
-Copy the content of the kubeconfig and insert it into
-`~/.kube/config`. Create the directory and the file if needed.
+## Extracting the kubeconfig file
 
-> By default, Kubectl searches for its configuration in
-> `~/.kube/config`, but you can modify this behaviour if needed. More
-> info
-> [here](https://kubernetes.io/docs/concepts/configuration/organize-cluster-access-kubeconfig/).
+When the Shoot cluster is up and running, you need to create a [kubeconfig](https://kubernetes.io/docs/concepts/configuration/organize-cluster-access-kubeconfig/) file to be able to access it.
+To do that, click on the cluster to expand its properties, and open *KubeConfig*.
 
-Check if your kubectl uses the proper configuration by running:
+![KubeConfig tab in {{k8s_management_service}} Shoot view](assets/shoot_kubeconfig.png)
+
+Copy the content of the kubeconfig using the *Copy Config* button, and insert it into `~/.kube/config`.
+Create the directory and the file if needed.
+
+> By default, `kubectl` searches for its configuration in `~/.kube/config`, but you can [modify this behaviour](https://kubernetes.io/docs/concepts/configuration/organize-cluster-access-kubeconfig/) if needed.
```

Comment:
```suggestion
> By default, `kubectl` searches for its configuration in `~/.kube/config`, but you can [modify this behavior](https://kubernetes.io/docs/concepts/configuration/organize-cluster-access-kubeconfig/) if needed.
```

Reply:
We had the 🇬🇧/🇨🇦 spelling in the original, but I did check the rest of the docs and we already do use 🇺🇸 spelling for this word in two other places. So I'll fix this.